src/OpenLoco/Localisation/StringIds.h

```diff
@@ -6,1819 +6,1820 @@
-    constexpr string_id cheat_ratings_to_min = 2207;
-    constexpr string_id cheat_ratings_to_max = 2208;
-    constexpr string_id all_vehicles = 2209;
-    constexpr string_id stopping_at_station = 2210;
-    constexpr string_id transporting_cargo = 2211;
-    constexpr string_id carrying_cargoid_sprite = 2212;
-    constexpr string_id carrying_cargoid_sprite_selected = 2213;
-    constexpr string_id no_station_selected = 2214;
-    constexpr string_id no_cargo_selected = 2215;
-    constexpr string_id tooltip_open_station_window_to_filter = 2216;
-    constexpr string_id tooltip_select_cargo_type = 2217;
-    constexpr string_id shortcut_construction_previous_tab = 2218;
-    constexpr string_id shortcut_construction_next_tab = 2219;
-    constexpr string_id shortcut_construction_previous_track_piece = 2220;
-    constexpr string_id shortcut_construction_next_track_piece = 2221;
-    constexpr string_id shortcut_construction_previous_slope = 2222;
-    constexpr string_id shortcut_construction_next_slope = 2223;
-    constexpr string_id shortcut_construction_build_at_current_pos = 2224;
-    constexpr string_id shortcut_construction_remove_at_current_pos = 2225;
-    constexpr string_id shortcut_construction_select_position = 2226;
-    constexpr string_id trainsReverseAtSignals = 2227;
-    constexpr string_id loan_autopay = 2228;
-    constexpr string_id tooltip_loan_autopay = 2229;
-    constexpr string_id cheat_year = 2230;
-    constexpr string_id cheat_month = 2231;
-    constexpr string_id cheat_day = 2232;
-    constexpr string_id cheat_date_change_apply = 2233;
-    constexpr string_id cheat_year_value = 2234;
-    constexpr string_id cheat_month_value = 2235;
-    constexpr string_id cheat_day_value = 2236;
-    constexpr string_id date_daymonthyear = 2237;
-    constexpr string_id shortcut_game_speed_normal = 2238;
-    constexpr string_id shortcut_game_speed_fast_forward = 2239;
-    constexpr string_id shortcut_game_speed_extra_fast_forward = 2240;
-    constexpr string_id vehicle_capacity_stringid = 2241;
-    constexpr string_id stats_obsolete = 2242;
-    constexpr string_id vehicle_is_locked = 2243;
-    constexpr string_id display_locked_vehicles = 2244;
-    constexpr string_id tooltip_display_locked_vehicles = 2245;
-    constexpr string_id stats_proposed_design = 2246;
-    constexpr string_id allow_building_locked_vehicles = 2247;
-    constexpr string_id tooltip_build_locked_vehicles = 2248;
-    constexpr string_id cheat_build_vehicle_window = 2249;
-    constexpr string_id tooltip_paint_landscape_tool = 2250;
-    constexpr string_id invert_right_mouse_dragging = 2251;
-    constexpr string_id tooltip_invert_right_mouse_dragging = 2252;
-    constexpr string_id cash_popup_rendering = 2253;
-    constexpr string_id tooltip_cash_popup_rendering = 2254;
-    constexpr string_id startServer = 2255;
-    constexpr string_id closeServer = 2256;
-    constexpr string_id disconnect = 2257;
+	constexpr string_id empty = 0;
```

Comment:
God damn, I haven't changed anything :( stupid visual studio. I literally have clang format running on file save so the only way this can happen is if git is merging it and converting them to tabs, I'll have to change that setting.
Will fix soon